hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -144,20 +145,19 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       const dataSize = duResponse.size;
       let subpaths: IDUSubpath[] = duResponse.subPaths;
 
-      subpaths.sort((a, b) => (a.size < b.size) ? 1 : -1);
-
-      // Only show top n blocks with the most DU,
-      // other blocks are merged as a single block
-      if (subpaths.length > limit) {
+      // This logic is for other objects for Max Limit 30 we are calculating 
+      // Other Object Size = Total Size - Sum of all subpaths Size
+      if (subpaths.length >= limit || (subpaths.length > 0 && limit === MAX_DISPLAY_LIMIT)) {
```

Comment:
Why do we need the `>=` in the `subpaths.length >= limit` condition? If we have 10 subpaths and 10 is the limit, we won't need "Other objects", right?

Reply:
@dombizita  Other Objects are depending on Total size and sum of all subpaths Size.
Example 1 
Limit=10 and subpaths.length=10
Total Size =100KB Sum of all 10 nodes size=90KB then we need to show remaining 10 in other objects to cover this test case checking condition  >=

Reply:
In your example the `subpaths.length` is 10. In that case the sum of that can't be less, than the total. The subpaths's size is either the total size or if we have 30 subpaths, we have the chance that we have more subpaths, in that case we need to sum them up and subtract it from the total size. 

If the `subpaths.length` is not the maximum limit, than we won't need other objects, unless the limit is less than the number of subpaths.

Reply:
Please correct me if I'm wrong @ArafatKhan2198 @devmadhuu

Reply:
@dombizita 
Done changes as subpaths.length > limit in latest commit.